### PR TITLE
NEWS: update document about WithoutRA=

### PR DESCRIPTION
**Pull request by yuwata**
Follow-up for 838d39af93e16e749ef90acf915a634b380abbf6.

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -178,12 +178,12 @@ CHANGES WITH 246 in spe:
...
 
-        * systemd-networkd's [DHCPv6] section gained a new WithoutRA= boolean
-          setting. If enabled, DHCPv6 will be attempted right-away without
-          requiring an Router Advertisement packet suggesting it
-          first. Conversely, the [IPv6AcceptRA] gained a boolean option
-          DHCPv6Client= that may be used to turn off the DHCPv6 client even if
-          the RA packets suggest it.
+        * systemd-networkd's [DHCPv6] section gained a new WithoutRA= setting.
+          If enabled, DHCPv6 will be attempted right-away without requiring an
+          Router Advertisement packet suggesting it first. Conversely, the
```
Comment (ssahani) [inline NEWS:183]: May be "Router Advertisement packet suggesting (i.e. neither 'M' nor 'O' flags are set) it.

File: NEWS [hunk 2]
```diff
@@ -178,12 +178,12 @@ CHANGES WITH 246 in spe:
...
+          [IPv6AcceptRA] section gained a boolean option DHCPv6Client= that may
+          be used to turn off the DHCPv6 client even if the RA packets suggest
```
Comment (keszybz) [inline NEWS:185]: I think "boolean" was making the whole clearer.

Comment (yuwata) [reply NEWS:185]: Sorry, I cannot follow the above comments by @ssahani and @keszybz. Anyway, we usually merge commits about NEWS and provide better words in later commits. Please submit in later commits with better explanation. Sorry for my bad explanation.

Comment (keszybz) [reply NEWS:185]: See a68da22257.

Comment (yuwata) [reply NEWS:185]: Ugh, WithoutRA= is not a boolean anymore...

**Pull request merged**